packages/entity/src/rules/PrivacyPolicyRuleEnums.ts

```diff
@@ -0,0 +1,61 @@
+/**
+ * Enum return type of privacy policy rule evaluation.
+ */
+export enum RuleEvaluationResult {
+  /**
+   * Deny viewer access to the entity.
+   */
+  DENY = -1,
+
+  /**
+   * Defer entity viewer access to subsequent rule in the privacy policy.
+   */
+  SKIP = 0,
+
+  /**
+   * Allow viewer access to the entity.
+   */
+  ALLOW = 1,
+}
+
+/**
+ * Estimated complexity of a privacy policy rule. Definition of what rules fall into what complexity category
+ * is application-specific, but the AlwaysAllow/AlwaysDeny/AlwaysSkip rules provided are all defined as LOW
+ * complexity.
+ */
+export enum RuleComplexity {
```

Comment:
Last suggestion is to call this cost instead of complexity since complexity tends to imply O(?) but a privacy policy could be costly if it does a waterfall of data fetches even if it runs in O(1).